app/src/main/java/vn/mbm/phimp/me/utils/FileUtils.java

```diff
@@ -0,0 +1,83 @@
+package vn.mbm.phimp.me.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+import android.content.Context;
+import android.text.format.Formatter;
+
+/**
+ * Created by rohanagarwal94 on 29/3/17.
+ */
+
+public class FileUtils {
+    private FileUtils() {
+
```

Comment:
@Abhi2424shek it is just the constructor of the class. Just trying to follow good coding practices.